test/support/src/vfs_helpers.cc

```diff
@@ -29,10 +29,22 @@
  *
  * This file defines some vfs-specific test suite helper functions.
  */
-#include "test/support/src/vfs_helpers.h"
+
+// The order of header includes is important here. serialization_wrappers.h
+// must be included before tdb_catch.h or anything that includes tdb_catch.h.
+// This is due to catch2 including Windows.h which defines things that break
```

Comment:
Shouldn't this be fixed by `https://github.com/catchorg/Catch2/pull/2526`? (which was included in catch > 3.1)